File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL7001Q01LayDailyReportInfo.java
package nta.med.data.model.ihis.cpls;

public class CPL7001Q01LayDailyReportInfo {
	private String gumsaDate;
    private String bunho;
    private String suname;
    private String gwa;
    private String doctorName;
    private String hangmogCode;
    private String hangmogName;
    private String specimenSer;
    private String specimenName;
	public CPL7001Q01LayDailyReportInfo(String gumsaDate, String bunho,
			String suname, String gwa, String doctorName, String hangmogCode,
			String hangmogName, String specimenSer, String specimenName) {
		super();
		this.gumsaDate = gumsaDate;
		this.bunho = bunho;
		this.suname = suname;
		this.gwa = gwa;
		this.doctorName = doctorName;
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
		this.specimenSer = specimenSer;
		this.specimenName = specimenName;
	}
	public String getGumsaDate() {
		return gumsaDate;
	}
	public void setGumsaDate(String gumsaDate) {
		this.gumsaDate = gumsaDate;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	public String getSpecimenSer() {
		return specimenSer;
	}
	public void setSpecimenSer(String specimenSer) {
		this.specimenSer = specimenSer;
	}
	public String getSpecimenName() {
		return specimenName;
	}
	public void setSpecimenName(String specimenName) {
		this.specimenName = specimenName;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL7001Q01LayDailyReportInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
    public class CPL7001Q01LayDailyReportInfo
    {
        private String _gumsaDate;
        private String _bunho;
        private String _suname;
        private String _gwa;
        private String _doctorName;
        private String _hangmogCode;
        private String _hangmogName;
        private String _specimenSer;
        private String _specimenName;

        public String GumsaDate
        {
            get { return this._gumsaDate; }
            set { this._gumsaDate = value; }
        }

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String HangmogName
        {
            get { return this._hangmogName; }
            set { this._hangmogName = value; }
        }

        public String SpecimenSer
        {
            get { return this._specimenSer; }
            set { this._specimenSer = value; }
        }

        public String SpecimenName
        {
            get { return this._specimenName; }
            set { this._specimenName = value; }
        }

        public CPL7001Q01LayDailyReportInfo() { }

        public CPL7001Q01LayDailyReportInfo(String gumsaDate, String bunho, String suname, String gwa, String doctorName, String hangmogCode, String hangmogName, String specimenSer, String specimenName)
        {
            this._gumsaDate = gumsaDate;
            this._bunho = bunho;
            this._suname = suname;
            this._gwa = gwa;
            this._doctorName = doctorName;
            this._hangmogCode = hangmogCode;
            this._hangmogName = hangmogName;
            this._specimenSer = specimenSer;
            this._specimenName = specimenName;
        }

    }
}